### DISCUSSION TITLE
URL supercell.com

### DISCUSSION THREAD

**Discussion by ITNetSystem** [Ground-Zerro/DomainMapper#48] [General]
Подскажи с каких источников ты формируешь адреса до сервисов?
Мне например нужны url адреса до supercell.com

**Comment by Ground-Zerro** [comment]
@ITNetSystem 
Интернет, поиск,  сообщества. Бывает кто-то уже собрал остается дополнить если есть чем.
По субдоменам [написал утилиты](https://github.com/Ground-Zerro/DomainMapper/tree/main/utilities). Одна собирает А субдомены, второй можно убрать припаркованные и не активыне субдомены. Правда источник бывает капризничает, но в целом собрать можно.
Пример их работы для supercell.com
[Активные домены.txt](https://github.com/user-attachments/files/17482512/default.txt)
[Полный список.txt](https://github.com/user-attachments/files/17482513/default.txt)
